From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@krist00fer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@krist00fer Thank you for this contribution.  I'll have the author review.

@davidmu1 Would you please review this change and let me know if I should merge it?

davidmu1 commented:
Please merge.

paulathurman committed:
edit pass: media-services-final-batch

paulathurman committed:
edit pass: media-services-final-batch

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into media-services-final-batch

dkamstra committed:
Fix a bug

paulathurman committed:
edit pass: media-services-final-batch

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into media-services-final-batch

paulathurman committed:
edit pass: media-services-final-batch

paulathurman committed:
edit pass: media-services-final-batch

paulathurman committed:
Update media-services-release-notes.md

mattbriggs committed:
adding a note and a switch

mattbriggs committed:
change warning to note

mattbriggs committed:
fix missing word

mattbriggs committed:
adding ujpdate to the content

paulathurman committed:
fixing merge conflicts

paulathurman committed:
Merge branch 'media-services-final-batch' of https://github.com/paulathurman/azure-docs-pr into media-services-final-batch

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

JeffGoldner committed:
Remove reference to eval image in marketplace and adjust screenshot

mmacy committed:
[aci] add mult-port expose note

mattbriggs committed:
adding current changes

mattbriggs committed:
adding item

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

CarlRabeler committed:
classic portal retirement

mattbriggs committed:
update  Operator Portal

CarlRabeler committed:
2

JeffGoldner committed:
Updated with information on the 1712 releases

nitinme committed:
freshness updates # 1

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

CarlRabeler committed:
minor fixes

alkohli committed:
Updated links to classic portal

alkohli committed:
Updates links to classic portal

alkohli committed:
Updated links to classic portal

alkohli committed:
Updated links to classic portal

alkohli committed:
Updated links to classic portal

alkohli committed:
Updated links to classic portal

alkohli committed:
Updated links to classic portal

alkohli committed:
Updated links to classic portal

seanli1988 committed:
more settings

Lisaco88 committed:
Merge pull request #31155 from alkohli/patch-164

Updated links to classic portal

JeffGoldner committed:
fix per pr comments

Lisaco88 committed:
Merge pull request #31157 from alkohli/patch-165

Updated links to classic portal

nitinme committed:
more freshness updates

Lisaco88 committed:
Merge pull request #31160 from alkohli/patch-166

Updated links to classic portal

Lisaco88 committed:
Merge pull request #31161 from alkohli/patch-167

Updated links to classic portal

tdykstra committed:
"on-premises functions" in toc

JeffGoldner committed:
fix ? mark

seanli1988 committed:
line breaks

Lisaco88 committed:
Merge pull request #31162 from alkohli/patch-168

Updated links to classic portal

mmacy committed:
[aci] add multi-port expose to template

Lisaco88 committed:
Merge pull request #31141 from mattbriggs/mdb_node-actions

Add update for node actions

Lisaco88 committed:
Merge pull request #31164 from alkohli/patch-169

Updated links to classic portal

Lisaco88 committed:
Merge pull request #31165 from alkohli/patch-170

Updated links to classic portal

tfitzmac committed:
updated version info

Lisaco88 committed:
Merge pull request #31154 from JeffGoldner/master

update for new versions of RPs and fix the info about default image

CarlRabeler committed:
links

seanli1988 committed:
tweaking

mmacy committed:
[aci] aci-connector > virtual-kubelet

Lisaco88 committed:
Merge pull request #31156 from nitinme/nitinme-freshness-updates

Freshness updates # 1

CarlRabeler committed:
more links

ktoliver committed:
Merge pull request #31170 from tdykstra/patch-7

"on-premises functions" in toc

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

nitinme committed:
freshness updates # 2

ktoliver committed:
Merge pull request #31149 from CarlRabeler/mirek-aad

Mirek aad

mmacy committed:
[aci] multi-volume mount via template

ktoliver committed:
Merge pull request #31166 from alkohli/patch-171

Updated links to classic portal

Lisaco88 committed:
Made two minor fixes

Lisaco88 committed:
Minor edits

ktoliver committed:
Merge pull request #31168 from seanli1988/patch-2

more settings

mmacy committed:
[aci] revert change to container-groups

ktoliver committed:
Merge pull request #31176 from tfitzmac/0109fix

updated version info

ktoliver committed:
Merge pull request #31177 from mmacy/aci-connector-deprecate

[aci] aci-connector > virtual-kubelet

ktoliver committed:
Merge pull request #31179 from nitinme/nitinme-freshness-updates

Freshness updates # 2

Lisaco88 committed:
Merge pull request #30954 from mattbriggs/mbd_recovery20180105

adding a note and a switch

ktoliver committed:
Merge pull request #31143 from mmacy/aci-ports-multi

[aci] add multi-port expose note

mmacy committed:
[aci] add volumeMounts template def

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into media-services-final-batch

paulathurman committed:
edit pass: media-services-final-batch

nabhishek committed:
updated adf regions in example and DMG->SH-IR

bwren committed:
Remove OMS 2

dubansal committed:
Skipping FillInMissingValuesStep will not throw exception

Removed warning to user to not remove FillInMissingValuesStep. Now skipping FillInMissingValuesStep will not throw an exception.

Juliako committed:
Update api-management-page-templates.md

bwren committed:
solution file typo

mrbullwinkle committed:
Update TOC to include App Insights Samples

bwren committed:
acrolinx

Lisaco88 committed:
Merge pull request #31180 from mmacy/aci-multi-volume-mount

[aci] multi-volume mount via template

Lisaco88 committed:
Merge pull request #31187 from dubansal/patch-4

Skipping FillInMissingValuesStep will not throw exception

Lisaco88 committed:
Merge pull request #31188 from bwren/la-oms

Remove OMS from Log Analytics 2

Lisaco88 committed:
Merge pull request #31190 from bwren/oms-solution

OMS solution file typo

Juliako committed:
Update api-management-page-templates.md

Lisaco88 committed:
Merge pull request #31186 from nabhishek/patch-16

updated adf regions in example and DMG->SH-IR

Lisaco88 committed:
Merge pull request #31189 from Juliako/patch-1

Update api-management-page-templates.md

Commit:
updates

Commit:
update

Commit:
acrolinx fix

Commit:
updates to metadata

daveba committed:
update saas tutorial list

Commit:
Fixed URLs

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into saneppal/updatelinks

gsacavdm committed:
Fix code block formatting

dominicbetts committed:
Fix typo spotted by Hans

Commit:
Removed preview for augmented rules.

The feature has gone GA: https://azure.microsoft.com/en-us/updates/agumented-rules-ga-nsg/

Commit:
Removed dynamic IP potentially changing.

Commit:
A few dynamic IP adjustments.

veljko-msft committed:
Update sql-database-advisor.md

JeffGoldner committed:
update mysql connector to the correct version

JeffGoldner committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Commit:
Added CSP.

JeffGoldner committed:
updated build numbers

mahesh-unnikrishnan committed:
Update active-directory-ds-faqs.md

Lisaco88 committed:
Merge pull request #31191 from mrbullwinkle/mrb_01_09_2018

Update TOC to include App Insights Samples

Commit:
Merge pull request #30717 from dkamstra/master

Fix a bug

Lisaco88 committed:
Merge pull request #31194 from georgewallace/autotagupdate

auto : metadata update - 1

Lisaco88 committed:
Merge pull request #31195 from MicrosoftDocs/saas-app-tutorial-list-update

update saas tutorial list

Lisaco88 committed:
Merge pull request #31199 from nstulasi/saneppal/updatelinks

Saneppal/updatelinks

itechedit committed:
Merge pull request #31201 from gsacavdm/patch-4

Fix code block formatting

Lisaco88 committed:
Merge pull request #31202 from dominicbetts/bug-fixes

Fix typo in Connected Factory article

Lisaco88 committed:
Merge pull request #31204 from jimdial/patch-57

Removed dynamic IP potentially changing.

itechedit committed:
Update security-overview.md

Lisaco88 committed:
Merge pull request #31205 from jimdial/patch-58

A few dynamic IP adjustments.

itechedit committed:
Merge pull request #31203 from jimdial/patch-56

Removed preview for augmented rules.

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Lisaco88 committed:
Merge pull request #31206 from veljko-msft/patch-12

Update sql-database-advisor.md

ShawnJackson committed:
Merge pull request #30759 from paulathurman/media-services-final-batch

edit pass: Media services final batch

Lisaco88 committed:
Merge pull request #31207 from JeffGoldner/master

Update MySQL connector to the correct version

v-alje committed:
Merge pull request #31211 from MicrosoftDocs/FromPublicRepo

From public repo

Lisaco88 committed:
Merge pull request #31209 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-faqs.md
